src/coreclr/jit/lsraarm64.cpp

```diff
@@ -1981,30 +1981,7 @@ int LinearScan::BuildHWIntrinsic(GenTreeHWIntrinsic* intrinsicTree, int* pDstCou
             getLowVectorOperandAndCandidates(intrin, &lowVectorOperandNum, &lowVectorCandidates);
         }
 
-        if ((intrin.id == NI_Sve_ConditionalSelect) && (intrin.op2->IsEmbMaskOp()) &&
```

Comment:
This was dead code

Reply:
Yeah, looks like we handle embedded masks just above this snippet, right [here](https://github.com/dotnet/runtime/blob/66168148b535eac364fadc8374c5deda1a328ce9/src/coreclr/jit/lsraarm64.cpp#L1864).